uniprot-search/src/main/java/org/uniprot/store/search/SolrQueryUtil.java

```diff
@@ -32,7 +33,7 @@ public static String getTermValue(String inputQuery, String term) {
     public static List<String> getTermValues(String inputQuery, String term) {
         List<String> result = new ArrayList<>();
         try {
-            QueryParser qp = new QueryParser("", new StandardAnalyzer());
+            QueryParser qp = new QueryParser("", new WhitespaceAnalyzer());
             qp.setAllowLeadingWildcard(true);
```

Comment:
Hi Leo,
Will the WhitespaceAnalyser not return stop words also? I am not very sure about this new analyser.